server/blob/blob-cassandra/src/test/java/org/apache/james/blob/cassandra/cache/CachedBlobStoreTest.java

```diff
@@ -274,4 +279,37 @@ public void shouldNotCacheWhenReadWithBigStream() {
             assertThat(Mono.from(cache.read(blobId)).blockOptional()).isEmpty();
         });
     }
+
+    @Test
+    public void readBlobstoreCachedShouldPublishTimerMetrics() {
+        BlobId blobId = Mono.from(backend.save(DEFAULT_BUCKETNAME, APPROXIMATELY_FIVE_KILOBYTES, SIZE_BASED)).block();
+
+        testee.read(DEFAULT_BUCKETNAME, blobId);
+        testee.read(DEFAULT_BUCKETNAME, blobId);
+
+        assertThat(metricFactory.executionTimesFor(BLOBSTORE_CACHED_LATENCY_METRIC_NAME))
+            .hasSize(2);
+    }
+
+    @Test
+    public void readBlobstoreCachedShouldCountWhenHit() {
+        BlobId blobId = Mono.from(testee.save(DEFAULT_BUCKETNAME, APPROXIMATELY_FIVE_KILOBYTES, SIZE_BASED)).block();
+
+        testee.read(DEFAULT_BUCKETNAME, blobId);
+        testee.read(DEFAULT_BUCKETNAME, blobId);
+
+        assertThat(metricFactory.countFor(BLOBSTORE_CACHED_HIT_COUNT_METRIC_NAME))
+            .isEqualTo(2);
+    }
+
+    @Test
+    public void readBlobstoreCachedShouldCountWhenMissed() {
+        BlobId blobId = Mono.from(testee.save(DEFAULT_BUCKETNAME, APPROXIMATELY_FIVE_KILOBYTES, SIZE_BASED)).block();
+
+        testee.read(DEFAULT_BUCKETNAME, blobId);
+        testee.read(DEFAULT_BUCKETNAME, blobId);
+
+        assertThat(metricFactory.countFor(BLOBSTORE_CACHED_HIT_COUNT_METRIC_NAME))
```

Comment:
Should be missed metrics. Looks like a bad copy/paste mistake here, as it's exactly the test above :)

Reply:
It is a bad mistake for rush.